src/Controller/Admin/BlogController.php

```diff
@@ -54,10 +55,9 @@ class BlogController extends AbstractController
      * @Route("/", name="admin_post_index")
      * @Method("GET")
      */
-    public function index(): Response
+    public function index(PostRepository $postRepository): Response
```

Comment:
Maybe unrelated to this PR, but in other projects I see this practice: `PostRepository $postRepository` -> `PostRepository $posts`. Instead of `$xxxRepository`, they use the plural of `$xxx` as the variable name. E.g. `UserRepository $users`, `ProductRepository $products`, etc. What do you think?

Reply:
Yes, I like that and feels better! so done.